mk/compile.mk

```diff
@@ -268,7 +268,7 @@ cleanfiles := $$(cleanfiles) $2 \
 		$$(dtb-predts-$2) $$(dtb-predep-$2) \
 		$$(dtb-dep-$2) $$(dtb-cmd-file-$2)
 
-dtb-cppflags-$2	:= -Icore/include/ -x assembler-with-cpp \
+dtb-cppflags-$2 := -Icore/include/ -x assembler-with-cpp -Ulinux -Uunix  \
```

Comment:
One space is enough before `\`

Reply:
thanks @jforissier and comments are addressed.